draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1402,6 +1402,45 @@ Note that it is possible for a certificate to be logged more than once. If that
 is the case, the log MAY return more than one entry index. If the certificate is
 present in the log, then the log MUST return at least one entry index.
 
+## Retrieve Signed Tree Heads between Two Times
+
+GET https://\<log server>/ct/v2/get-sths
+
+Inputs:
+
+: start:
+  : an earlier NTP Time [RFC5905], measured in milliseconds since the epoch
+    (January 1, 1970, 00:00 UTC), ignoring leap seconds.
+
+  end:
+  : a later NTP Time [RFC5905], measured in milliseconds since the epoch
+    (January 1, 1970, 00:00 UTC), ignoring leap seconds.
+
+Outputs:
+
+: sths:
+  : an array of base64 encoded `TransItem` structures of type
+    `signed_tree_head_v2`, signed by this log.
+
+The `start` and `end` parameters SHOULD be within the range 0 <= x < `timestamp`
```

Comment:
If a parameter is optional, it's explicitly marked as optional.  e.g., see the description of the "second" input to "get-sth-consistency".